asmrun/arm.S

```diff
@@ -17,7 +17,7 @@
 
         .syntax unified
         .text
-#if defined(SYS_linux_eabihf) && defined(MODEL_armv6)
+#if (defined(SYS_linux_eabihf) || defined(SYS_netbsd)) && defined(MODEL_armv6)
```

Comment:
Well, we could have SYS_eabi and SYS_eabihf, but then there would be no way to distinguish Linux from BSD, which we may need in the future. 

That doesn't preclude a refactoring local to asmrun/arm.S:
```
    #if ....
    #define ABI hardfloat
    #else
    #define ABI eabi
    #endif
```
then test on ABI.